### PR TITLE
add instructions to run the website on mobile's chrome browser.

### LINKED ISSUE #580: badgeyay site not working/showing up on android's chrome browser.
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

actually site is not secured (https instead of http) and thus gets manipulated by `data saver` option on chrome in mobile device.
**Expected behavior:**

Working web app on mobile device.
**Steps to reproduce:**
open badgeyay.com on google chrome, on a mobile device.

### PR DESCRIPTION
**Pull request by rupav**
Fixes #580 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- instruction for working of badgeyay on mobile's chrome browser by disabling [data saver](https://support.google.com/chrome/answer/2392284?co=GENIE.Platform%3DAndroid&hl=en) option.

Note: If we shift to `https` from current `http`, disabling data saver option will no longer be needed. Also if you use `chrome incognito` too, you don't need to disable data saver!

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -17,7 +19,7 @@ You will need the following things properly installed on your computer.
 
 To install the app, run the following commands in a command line:
 
-* `git clone https://github.com/fossasia/badgeyay this repository
+* git clone https://github.com/fossasia/badgeyay this repository
```
Comment (vaibhavsingh97) [inline frontend/README.md]: Replace "git clone https://github.com/fossasia/badgeyay this repository" w/ `git clone https://github.com/fossasia/badgeyay`

Comment (rupav) [reply frontend/README.md]: Done

File: frontend/README.md [hunk 2]
```diff
@@ -17,7 +19,7 @@ You will need the following things properly installed on your computer.
 
 To install the app, run the following commands in a command line:
 
-* `git clone https://github.com/fossasia/badgeyay this repository
+* `git clone https://github.com/fossasia/badgeyay` this repository
```
Comment (vaibhavsingh97) [inline frontend/README.md]: `this repository` can  be removed

Comment (rupav) [reply frontend/README.md]: Thats in installation step... why to delete this line ?

Comment (abishekvashok) [reply frontend/README.md]: Since it's unneeded as the code is present. Or you can write: clone this repository. Then the code within ``

Comment (rupav) [reply frontend/README.md]: @vaibhavsingh97 @abishekvashok I don't know why, but I misunderstood your request completely in the first place. Now its fixed 👍

**Pull request merged**